### PR TITLE
Exposing Transforms.rotationMatrixFromPositionVelocity

### PR DESCRIPTION
**Pull request by easternmotors**
Also added tests for Transforms.rotationMatrixFromPositionVelocity

NOTE: I wasn't able to figure out how to get the tests to pass since `Transforms.rotationMatrixFromPositionVelocity` sometimes return 0 or -0 for it's 0 values. I ended up just using the `Matrix3.abs` function for now but if there is a better way to handle this let me know.

Handles bullet 3 from #8927

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @easternmotors!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/master/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by easternmotors** [comment]
Updated CHANGES.md

File: Specs/Core/TransformsSpec.js [hunk 1]
```diff
@@ -1716,6 +1716,77 @@ describe("Core/Transforms", function () {
     expect(returnedResult).toEqualEpsilon(expected, CesiumMath.EPSILON12);
   });
 
+  it("rotationMatrixFromPositionVelocity works without a result parameter", function () {
+    var matrix = Transforms.rotationMatrixFromPositionVelocity(
+      Cartesian3.UNIT_X,
+      Cartesian3.UNIT_Y
+    );
+    var expected = new Matrix3(0, 0, 1, 1, 0, 0, 0, 1, 0);
+    // using Matrix3.abs to fix the 0/-0 comparisons
```
Comment (mramato) [inline Specs/Core/TransformsSpec.js:1725]: You should be able to replace this with `toEqualEpsilon` throughout.  For example:

```js
expect(matrix).toEqualEpsilon(expected, CesiumMath.EPSILON14);
```

**Comment by mramato** [comment]
Thanks @easternmotors, just the one comment to address the abs usage and hopefully this will be good to go.  Thanks.

File: Specs/Core/TransformsSpec.js (hunk 1 above)
Comment (easternmotors) [reply Specs/Core/TransformsSpec.js]: Awesome that works. Makes me feel a lot better

**Comment by mramato** [comment]
Thanks!

**Pull request merged**